Documentation/network/servicemesh/gateway-api/gateway-api.rst

```diff
@@ -0,0 +1,46 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_gateway_api:
+
+*******************
+Gateway API Support
+*******************
+
+Cilium supports Gateway API v0.5.1 for below features:
+
+- `GatewayClass <https://gateway-api.sigs.k8s.io/api-types/gatewayclass/>`_
+- `Gateway <https://gateway-api.sigs.k8s.io/api-types/gateway/>`_
+- `HTTPRoute <https://gateway-api.sigs.k8s.io/api-types/httproute/>`_
+- `ReferenceGrant <https://gateway-api.sigs.k8s.io/api-types/referencegrant/>`_
+
+.. Note::
+
+    Even though Cilium only supports above resources, the current
+    implementation passes all the conformance tests in the Gateway
+    API suite.
```

Comment:
Not clear to me what this means (but I'm not familiar with the concepts here, in particular the Gateway API suite).

Reply:
yeah, this is a little bit confusing, so it's better just remove it.